pnpm-lock.yaml

```diff
@@ -34,7 +34,7 @@ importers:
       prettier: 2.4.1
       typescript: ^5.0.4
     dependencies:
-      '@darajs/prettier-config': link:../prettier-config
+      '@darajs/prettier-config': 1.6.0
```

Comment:
Something went wrong here and is using 1.6 instead of 1.5.4

Reply:
Turns out it was picking up an already released 1.6.0 version regardless. Switched from caret range to tilde range